improve the collation tables according AGC review
(should be backported to CLDR probably)

--- core/common/collation/zh.xml
+++ core.new/common/collation/zh.xml
@@ -10213,7 +10213,6 @@
 		</collation>
 		<collation type='unihan'>
 			<cr><![CDATA[
-				[import und-u-co-private-unihan]
 				[import zh-u-co-private-pinyin]
 				[reorder Hani Bopo]
 			]]></cr>
